change/@ni-nimble-tokens-a6408b9a-6f2d-4d38-97e3-b6537160321c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
@fredvisser just to confirm, are tabs and anchor tabs expected to be aligned with the button label font? I noticed they were accepted changes but I didn't double check figma. If that's expected then good to go 👍

Reply:
Oh wait, you already said that in the PR description, my bad. Looks good